stable/airflow/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 description: Airflow is a platform to programmatically author, schedule and monitor workflows
 name: airflow
-version: 0.14.0
```

Comment:
That’s a new feature, actually. You can bump to 0.15.0

Reply:
Fixed.

Reply:
thanks!